test/any/client/api_matrix-test.es6.js

```diff
@@ -70,7 +70,13 @@ describe('ApiMatrix', function() {
       browserApiDAO,
       browserDAO,
       interfaceDAO,
-    });
+    },
+    // Mock exports expected from ApiExtractor.
```

Comment:
nit: We won't actually get them from `ApiExtractor`. This could say something like `Provide a context that is aware to relationship DAOs.` You could also add a `TODO(markdittmer)` about providing an interface for this on `Relationship`s.